src/errors.js

```diff
@@ -66,3 +66,34 @@ export const AccountAwaitingSendPendingOperations = createCustomErrorClass(
 );
 
 export const SourceHasMultiSign = createCustomErrorClass("SourceHasMultiSign");
+
+export const CosmosTooMuchRedelegations = createCustomErrorClass(
+  "CosmosTooMuchRedelegations"
+);
+export const CosmosTooMuchUnboundings = createCustomErrorClass(
+  "CosmosTooMuchUnboundings"
+);
+export const CosmosRedelegationInProgress = createCustomErrorClass(
+  "CosmosRedelegationInProgress"
+);
+
+export const CosmosBroadcastError = {
```

Comment:
Add the source for error codes so that if something weird happens we know where to look : https://github.com/cosmos/cosmos-sdk/blob/v0.37.9/types/errors.go#L28

I'm afraid that these values will have to be shifted by 1 when cosmos pushes the `protobuf` changes

Reply:
Those can be seen in a traditional user path
```
CodeOK                CodeType = 0
	CodeUnknownAddress    CodeType = 9
	CodeOutOfGas          CodeType = 12
	CodeMemoTooLarge      CodeType = 13
```

These are (probably) happening if we have a bug on Live side
```
	CodeTxDecode          CodeType = 2
	CodeInvalidSequence   CodeType = 3
	CodeUnauthorized      CodeType = 4
	CodeInsufficientFunds CodeType = 5
	CodeInvalidAddress    CodeType = 7
	CodeInsufficientCoins CodeType = 10
	CodeInsufficientFee   CodeType = 14
	CodeTooManySignatures CodeType = 15
	CodeGasOverflow       CodeType = 16
	CodeNoSignatures      CodeType = 17
```

These probably happen if we have an issue on Core side
```
	CodeUnknownRequest    CodeType = 6
	CodeInvalidPubKey     CodeType = 8
	CodeInvalidCoins      CodeType = 11
```

This one means we have a serious issue (in that, we don't know what happens)
```
CodeInternal          CodeType = 1
```

Basically we have this information coming directly from the underlying node we use for broadcasting, and if users have issues sending transaction, having the exact value in the "Screenshot" that users send is going to be a _huge_ help to debug the issue. Maybe we can "skip" traduction by saying it's an "internal error" with the english name for the code ?

Reply:
ok. i guess we can keep these error here, and this comment will be good to guide to what worth a translation.